package.json

```diff
@@ -64,13 +77,15 @@
   "devDependencies": {
     "babel-runtime": "5.8.29",
     "babelify": "6.4.0",
-    "browserify": "6.1.0",
+    "browserify": "10.0.0",
+    "browserify-derequire": "^0.9.4",
```

Comment:
After a couple hard-to-debug cases where patch/minor upgrades had different behavior, I eventually started pegging all packages to exact versions for consistency (e.g. anyone building on any machine at any time should get the same thing). What do you think?